docs/quick-starts/bandit-quickstart.md

```diff
@@ -0,0 +1,188 @@
+---
+slug: /bandit-quickstart
+sidebar_position: 5
+---
+
+# Running your first Contextual Bandit
+
+This 10 minute guide will walk through configuring and running your first Contextual Bandit with Eppo. Note that Contextual Bandits leverage the Eppo SDK and this guide will walk through SDK setup.
+
+:::info
+In order to optimize a Bandit, you must have [created your first metric](/metric-quickstart).
```

Comment:
Would a reader know that “optimize” here means “run” a Bandit, or could they assume it means to improve an existing Bandit setup?